TL improvements (#327)
07th-mod/higurashi-console-arcs: Update/omot_012.txt

//rずいっ、と身を乗り出して、魅音がこわい顔で睨んでくる。その手には真新しいトランプが二枚、背を向けて俺にさっさと引け！　とばかりに突き出されていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ずいっ、と身を乗り出して、魅音がこわい顔で睨んでくる。その手には真新しいトランプが二枚、背を向けて俺にさっさと引け！　とばかりに突き出されていた。",

> 			NULL, "Mion leaned forward and glared at me with a terrifying look on her face. She was holding two brand-new playing cards in her hand, with their backs turned toward me, like saying just draw one!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mion leaned forward and glared at me with a terrifying look on her face. She was holding two brand-new playing cards in her hand, with their backs turned toward me, like telling me to draw one!", Line_Normal);
```

//r実はそれがハチだったことに気がついて必死に逃げ回り、命からがら疲労困憊で戻ってきて轟沈した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "実はそれがハチだったことに気がついて必死に逃げ回り、命からがら疲労困憊で戻ってきて轟沈した。",

> 			NULL, "After he realized it was actually a wasp, he continued to flee from it desperately. And when he finally came back, he immediately sank to the ground.", Line_Normal);

Reviewer: ```suggestion
			NULL, "When he realized it was actually a wasp, he ran like his life depended on it. And when he finally came back, he immediately sank to the ground.", Line_Normal);
```

//r…誰だよ、こんなミッション・インポッシブル考えたのは？！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…誰だよ、こんなミッション・インポッシブル考えたのは？！",

> 			NULL, "...Who thought up that one like Mission:Impossible?!", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Who thought up that Mission:Impossible scenario?!", Line_Normal);
```

			NULL, "\"Oh my, that barking dog voice feels so lovely~!", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300486", 256, TRUE);
	OutputLine(NULL, "　…あ、ちなみに私の書いた罰ゲームはまだみたいだから、引いたら頑張ってね！」",

> 			NULL, " ...Oh by the way, I still haven't seen the punishment I wrote, so do your best if you grab that one!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...Oh by the way, I still haven't seen the punishment I wrote, so good luck if you grab that one!\"", Line_Normal);
```

//r聞いたこともない料理だ。…ちなみにbやごうち.<谷河内>は、ダム工事の時には採石場があったところの地名。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "聞いたこともない料理だ。…ちなみに谷河内(やごうち)は、ダム工事の時には採石場があったところの地名。",

> 			NULL, "That's a dish I'd never heard of. ...Come to think of it, the quarry for the dam construction project was located in a village called Yagouchi.", Line_Normal);

Reviewer: ```suggestion
			NULL, "That's a dish I'd never heard of. ...Come to think of it, the quarry for the dam construction project was located in a place called Yagouchi.", Line_Normal);
```

//r梨花ちゃんが優しく、都会の人にもわかるように解説してくれる。どういうわけかカメラ目線だが、そんな補足いらん。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "梨花ちゃんが優しく、都会の人にもわかるように解説してくれる。どういうわけかカメラ目線だが、そんな補足いらん。",

> 			NULL, "Rika-chan was kind enough to explain it for everyone around us. She said it like she was looking into a camera, while I didn't need that kind of a footnote.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Rika-chan was kind enough to explain it for everyone around us. She said it like she was looking into a camera, but I could've done without that kind of a footnote.", Line_Normal);
```

> 			NULL, "Rika-chan was kind enough to explain it for everyone around us. She said it like she was looking into a camera, while I didn't need that kind of footnote.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Rika-chan was kind enough to explain it for everyone around us. She said it like she was looking into a camera, but I could've done without that kind of footnote.", Line_Normal);
```

			NULL, "\"What are you talking about?", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/210300493", 256, TRUE);
	OutputLine(NULL, "　イナゴとかハチの子とかの料理、前に出した時は圭ちゃんも『なかなか』って食べてたじゃん。",

> 			NULL, " I've brought food that contained grasshoppers and hornet pupa before, Kei-chan, and you ate it up saying <i>actually rather good</i>.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I've brought food that contained grasshoppers and hornet pupa before, Kei-chan, and you ate it up saying it was <i>actually rather good</i>.", Line_WaitForInput);
```

> 			NULL, " I've brought food that contained grasshoppers and hornet pupa before, Kei-chan, and you ate it up with a <i>rather good</i> remark.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I've brought food that contained grasshoppers and hornet pupa before, Kei-chan, and you ate it up, saying it was <i>rather good</i>.", Line_WaitForInput);
```

			NULL, "\"...Zzz~...\"", Line_Normal);
	ClearMessage();

//rとりあえず落ち着いたのか、穏やかな寝息を立てている魅音。…鼻血の跡がちょっとコミカルだが、一応女の子なので見なかったことにしてやろう。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "とりあえず落ち着いたのか、穏やかな寝息を立てている魅音。…鼻血の跡がちょっとコミカルだが、一応女の子なので見なかったことにしてやろう。",

> 			NULL, "At least for now, things seemed to have calmed down with Mion peacefully asleep. ...It was kind of amusing to see her with a stain of nosebleed, but since she's a girl, I'll pretend I never saw it.", Line_Normal);

Reviewer: ```suggestion
			NULL, "At least for now, things seemed to have calmed down with Mion peacefully asleep. ...It was kind of amusing to see her with a nosebleed stain, but since she's a girl, I'll pretend I never saw it.", Line_Normal);
```

//r…梨花ちゃんの話だと、この前の世界…敵が誰かが判明した世界の記憶について、俺は引き継いでいないそうだ。だから敵があの看護師の鷹野さん、といわれてもいまひとつぴんと来ないのだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…梨花ちゃんの話だと、この前の世界…敵が誰かが判明した世界の記憶について、俺は引き継いでいないそうだ。だから敵があの看護師の鷹野さん、といわれてもいまひとつぴんと来ないのだ。",

> 			NULL, "...According to Rika-chan... I still haven't recovered my memories of the previous world, where we learned the identity of our enemy. So, even though she told me the enemy is a nurse: Takano-san, nothing has come back to me.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...According to Rika-chan... I still haven't recovered my memories of the previous world, where we learned the identity of our enemy. So, even though she told me the enemy is that nurse, Takano-san, nothing has come back to me.", Line_Normal);
```

	//rでも、…『東京』という組織に、入江機関。そして、雛見沢症候群という風土病のこと…。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "でも、…『東京』という組織に、入江機関。そして、雛見沢症候群という風土病のこと…。",

> 				NULL, "But... there's an organization called \"Tokyo\" connected to Irie Institute. And some endemic disease called Hinamizawa Syndrome...", Line_Normal);

Reviewer: ```suggestion
				NULL, "But... there's an organization called \"Tokyo\" connected to the Irie Institute. And some endemic disease called Hinamizawa Syndrome...", Line_Normal);
```

//r人の言動をもとに、その意図とかを色々と考えることは、楽しい。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "人の言動をもとに、その意図とかを色々と考えることは、楽しい。",

> 			NULL, "It's fun to think about the intent behind what someone talks about.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It's fun to think about the intent behind people's words and actions.", Line_Normal);
```

//rそう言って、魅音はふてくされたように顔を背けてしまう。…やれやれ、と俺は肩をすくめて、ベッドに歩み寄った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう言って、魅音はふてくされたように顔を背けてしまう。…やれやれ、と俺は肩をすくめて、ベッドに歩み寄った。",

> 			NULL, "Mion sulkily turned away as she said that. ...I shrugged my shoulders with a feeling of good grief and walked over to the bed.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mion sulkily turned away as she said that. ...I shrugged my shoulders in exasperation and walked over to the bed.", Line_Normal);
```

			NULL, "\"Don't take entrance exams lightly, all right?", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101157", 256, TRUE);
	OutputLine(NULL, "　ポイント絞ってやらなきゃ下手すりゃ全教科書けませんでした、無条件降伏。",

> 			NULL, " She couldn't hardly write answers in any and every subject, she just surrendered unconditionally, so she should have hammered away  selectively.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " You have to squeeze the points out of your textbooks, on every single subject, otherwise you're just surrendering unconditionally.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101158", 256, TRUE);
	OutputLine(NULL, "…なんてことがあるんだからな」",

> 			NULL, " ...That's a saying about ordeal.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " ...It's really an ordeal.\"", Line_Normal);
```

			NULL, "\"The Mion Sonozaki I know is invincible, undefeated! Far too great to be rivaled!", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210101160", 256, TRUE);
	OutputLine(NULL, "　それなのに勉強しか能のないガリベン野郎どもにたかが受験程度で負けて帰ってくるなんて、俺は絶対に認めねぇぞ！」",

> 			NULL, " That's why I'll never believe you come home defeated by grinds with the only ability to study in a mere entrance exam!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " That's why I'll never accept you coming home defeated by bastards who can't do anything but grind away studying for entrance exams!\"", Line_Normal);
```

//r魅音はぷい、と顔を戻して、それからかすかに身じろぎする。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "魅音はぷい、と顔を戻して、それからかすかに身じろぎする。",
			NULL, "Mion abruptly turned back this way, then stirred slightly.", Line_Normal);
	ClearMessage();

//r…なんとなく、頭が動いたときにこくん、と頷いたように見えたけど、それ以上追求するのは止めた。わざわざ確かめなくても、きっと魅音なら来てくれるだろう。そう思う。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…なんとなく、頭が動いたときにこくん、と頷いたように見えたけど、それ以上追求するのは止めた。わざわざ確かめなくても、きっと魅音なら来てくれるだろう。そう思う。",

> 			NULL, "...Somehow, it seemed like she started to nod when she turned this way, but I didn't dare to ask her any further. Even if I don't confirm it here, I'm sure Mion will come. At least, I think so.", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Somehow, it seemed like she started to nod when she turned this way, but I didn't dare to ask her any more. Even if she doesn't confirm it here, I'm sure Mion will come. At least, I think so.", Line_Normal);
```

//r魅音はそう言って、ひらひらと笑顔で手を振る。とりあえず俺はまあいいか、とばかりに小首を傾けてから、扉を開いて廊下に出た。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "魅音はそう言って、ひらひらと笑顔で手を振る。とりあえず俺はまあいいか、とばかりに小首を傾けてから、扉を開いて廊下に出た。",

> 			NULL, "Mion waved her arms as she said that and smiled. For the time being, I just tilted my head instead of saying Whatever, then opened the door and went out into the hallway.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Mion waved her arms as she said that and smiled. For the time being, I just tilted my head instead of saying anything, then opened the door and went out into the hallway.", Line_Normal);
```
